### ISSUE TITLE
Identify tags to labels from huggingface model

### ISSUE THREAD

**Issue by rattle99** [Sripaad/ai4privacy#2]
How does one identify which are the corresponding labels of the model on HF [link](https://huggingface.co/Isotonic/distilbert_finetuned_ai4privacy_v2) ?

Using the model and tokenizer available on HF. `out = model(**tokenized_text)`
It seems like there are 111 logits in the output dimension?

**Comment by rattle99** [comment]
Found the `id2label` mapping in `model.config.id2label`. I realized that the extra logits are for I-tokens that come in between.
